docs/release_notes/release_notes-0.16.0.rst

```diff
@@ -55,6 +55,8 @@ Key Features and Updates
   * FIX-#4996: Evaluate BenchmarkMode at each function call (#4997)
   * FIX-#4022: Fixed empty data frame with index (#4910)
   * FIX-#4090: Fixed check if the index is trivial (#4936)
+  * FIX-#4818, PERF-#4825: Fix where by using the new n-ary operator (#4820)    
```

Comment:
this change seems to be wrong, adding more things than it should

Reply:
Sorry, one more line added incidentally, after a merge conflict resolution.
Thanks for noticing.